_topic_map.yml

```diff
@@ -238,6 +238,13 @@ Topics:
   - Name: AWS Service Broker
     File: aws_broker
 ---
+Name: Backing up applications and projects
+Dir: admin_guide
+Distros: openshift-online
+Topics:
+  - Name: Backing up and restoring applicatons and projects
```

Comment:
I had mentioned this in slack, but this and line 246 shouldn't be indented in this far. It should be 2 columns to the left.

Reply:
You did, but you didn't say anything about pushing it. :-) It should be visible now.